assets/shaders/fs_object.sc

```diff
@@ -3,21 +3,21 @@ $input v_position, v_texcoord0, v_normal
 #include <bgfx_shader.sh>
 
 SAMPLER2D(s_diffuse, 0);
-
-uniform vec4 u_sky;
+uniform vec4 u_skyAlphaThreshold;
```

Comment:
I'm getting
`[2022-01-18 18:26:47.590] [graphics] [warning] [ShaderProgram.cpp:80] Could not find uniform u_skyAlphaThreshold`.  I think it's being optimised out.

Reply:
Oh yeah because sky is no longer using the object shader